strings/base_coroutine_threadpool.h

```diff
@@ -1,63 +1,108 @@
 
 namespace winrt::impl
 {
+    inline auto submit_threadpool_callback(void(__stdcall* callback)(void*, void* context), void* context)
+    {
+        if (!WINRT_IMPL_TrySubmitThreadpoolCallback(callback, context, nullptr))
+        {
+            throw_last_error();
+        }
+    }
+
     inline void __stdcall resume_background_callback(void*, void* context) noexcept
     {
         std::experimental::coroutine_handle<>::from_address(context)();
     };
 
     inline auto resume_background(std::experimental::coroutine_handle<> handle)
     {
-        if (!WINRT_IMPL_TrySubmitThreadpoolCallback(resume_background_callback, handle.address(), nullptr))
-        {
-            throw_last_error();
-        }
+        submit_threadpool_callback(resume_background_callback, handle.address());
     }
 
-    inline bool is_sta() noexcept
+    inline std::pair<int32_t, int32_t> get_apartment_type() noexcept
```

Comment:
consider mentioning the results hold `APTTYPE_*` and `APTTYPEQUALIFIER_*` values

Reply:
Already mentioned in the PR description.